view/frontend/web/js/components/steps/ThePaymentStep.vue

```diff
@@ -50,6 +50,7 @@
       class="button"
       button-type="button"
       text="Place order"
+      with-loader="true"
```

Comment:
you can change with-loader="true" to with-loader

Reply:
ok